k8s/overlays/azure-commons/case-server-application.yml

```diff
@@ -0,0 +1,30 @@
+server:
+  port: 80
+
+spring:
+  servlet:
+    multipart:
+      max-file-size: 100MB
+      max-request-size: 103MB
+
+  rabbitmq:
+    addresses: rabbitmq
+    port: 5672
+  cloud:
+    function:
+      definition: publishCaseImport
+    stream:
+      bindings:
+        publishCaseImport-out-0:
+          destination: case.import
+
+  data:
+    elasticsearch:
+      enabled : true
+      host: elasticsearch
+      port: 9200
+      index: case-server
+      type: metadatas
+
+case-store-directory: /cases
+max-request-size: 10000
```

Comment:
Missing PR in case-server repository (corresponding to branch 'limit_maximum_number_of_cases_in_case_server ?)

I didn't found usage of these new properties 'case-store-directory' and 'max-request-size' in this branch

Reply:
https://github.com/powsybl/powsybl-case/pull/58/